Upstream-Status: Inappropriate [configuration, from other distro Debian ]

## default_automaster_location.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Set /etc/auto.master as the default location of the automounter
## DP: master map, because that file is surely available (the package
## DP: installs a default one). If the admin already has an auto.master
## DP: map in a different place, he can customize the file later.


--- a/samples/autofs.conf.default.in
+++ b/samples/autofs.conf.default.in
@@ -3,7 +3,7 @@
 #
 # MASTER_MAP_NAME - default map name for the master map.
 #
-#MASTER_MAP_NAME="auto.master"
+MASTER_MAP_NAME="/etc/auto.master"
 #
 # TIMEOUT - set the default mount timeout (default 600).
 #
